ftw/copymovepatches/testing.py

```diff
@@ -16,11 +17,16 @@ def setUpZope(self, app, configurationContext):
             '  <include package="z3c.autoinclude" file="meta.zcml" />'
             '  <includePlugins package="plone" />'
             '  <includePluginsOverrides package="plone" />'
+            '  <include package="ftw.copymovepatches.tests" file="profiles.zcml" />'
             '</configure>',
             context=configurationContext)
 
     def setUpPloneSite(self, portal):
-        pass
+        z2.installProduct(app, 'ftw.copymovepatches')
```

Comment:
@maethu seems you've removed that? the `initialize` method is still there. Should we remove both or keep both?